GNUmakefile

```diff
@@ -257,12 +251,18 @@ hclfmt:
 	@find . -path ./terraform -prune -o -name 'upstart.nomad' -prune -o \( -name '*.nomad' -o -name '*.hcl' \) -exec \
 sh -c 'hclfmt -w {} || echo in path {}' ';'
 
+.PHONY: sync
+sync:
+	@echo "--> Sync vendor directory and tidy up"
+	@go mod vendor
+	@go mod tidy
```

Comment:
You may want to also tidy any submodules in an order that makes sense, like:
```
@cd tools; go mod tidy
@cd api ; go mod tidy
@go mod tidy
@go mod vendor
```

Reply:
Good idea, thanks!